### DISCUSSION TITLE
Support GDB debug format

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#991] [Ideas]
VS is too heavy and as I believe most hobbyists like me can't afford running it on our ancient computers. Please consider. Thank you.

#271

**Comment by GitMensch** [comment]
You likely mean DWARF. Note that there are a bunch of debugging tools that understand PDB, you don't need VS for that. WinDBG is possibly the best known one (it is also on the MS Store), also LMDB supports that on Windows (which for example allows you to use lmdb-mi to debug in vscode or other editors with MI support)

Just saying, I do prefer DWARF, too, as it is better portable to non Win32 environments and there are also a lot of tools available for it.

**Comment by LADSoft** [comment]
ok maybe i will prioritize this over the VS solution then...
